packages/ra-keycloak/Readme.md

```diff
@@ -0,0 +1,92 @@
+# ra-keycloak
```

Comment:
You should not duplicate the content in `README.md` and in `packages/ra-keycloak/Readme.md`, because we may maintain one and forget to maintain the other. Make the root readme reference or include this one.